src/pfe/portal/modules/User.js

```diff
@@ -80,6 +80,15 @@ module.exports = class User {
 
       // Create the list of codewind extensions
       this.extensionList = new ExtensionList();
+
+      // Attempt to install built-in extension packages
+      try {
+        await this.extensionList.installBuiltInExtensions(this.directories.extensions);
+      } catch (error) {
+        log.error('Fail to install built-in Codewind extensions');
```

Comment:
nit: can we have the 2 errors in one entry, to make the log slightly clearer? i.e. 
```suggestion
        log.error(`Failed to install built-in Codewind extensions. Error ${util.inspect(error)}`);
```

Reply:
And remove `log.error(error)` below?